tests/crowdsourcing/tasks/test_chat_demo.py

```diff
@@ -386,6 +386,7 @@ def test_base_task(self):
                 '++mephisto.task.maximum_units_per_worker=0',
                 '++num_turns=3',
                 '++turn_timeout=300',
+                '++mephisto.task.submission_timout=10',
```

Comment:
Hmm what's the reason for this change?

Reply:
we added a new config option here that I wasn't able to get hydra's configure to pick up the default of. This was the first place I could find that restored it.

Reply:
Fair enough (nit: typo in the var name I believe)

Reply:
> Fair enough (nit: typo in the var name I believe)

_that_ was the real issue, thanks for the catch! Pushing this through the 1.0 release branch right now

Reply:
Ah great - hmm in that case, presumably we don't still need this line in the PR once the Hydra flag name is changed?

Reply:
We'll see once I get 1.0 out and we're pulling from there instead.

Reply:
Just removed these lines